packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -29,6 +30,12 @@ export class Snackbar implements ComponentInterface {
    */
   @Prop() inoAlignment?: 'left' | 'right' | 'center' = 'center';
 
+  /**
+   * Changes the snackbar type
+   * Valid options are: warning, error
+   */
+  @Prop() inoType?: SnackbarType;
```

Comment:
If we use a literal type, we should provide a default value. Maybe add another option `default` which is the default value of `inoType` (analogous to line 31).

Reply:
✅